src/tests/stub_comm.cc

```diff
@@ -16,55 +16,58 @@
 
 #include <ostream>
 
-void comm_read(const Comm::ConnectionPointer &conn, char *buf, int size, IOCB *handler, void *handler_data) STUB
-void comm_read(const Comm::ConnectionPointer &conn, char*, int, AsyncCall::Pointer &callback) STUB
+void comm_read(const Comm::ConnectionPointer &, char *, int, IOCB *, void *) STUB
+void comm_read(const Comm::ConnectionPointer &, char*, int, AsyncCall::Pointer &) STUB
 
 /* should be in stub_CommRead */
 #include "CommRead.h"
 CommRead::CommRead(const Comm::ConnectionPointer &, char *, int, AsyncCall::Pointer &) STUB
 CommRead::CommRead() STUB
 DeferredReadManager::~DeferredReadManager() STUB
 DeferredRead::DeferredRead(DeferrableRead *, void *, CommRead const &) STUB
-void DeferredReadManager::delayRead(DeferredRead const &aRead) STUB
-void DeferredReadManager::kickReads(int const count) STUB
+void DeferredReadManager::delayRead(DeferredRead const &) STUB
+void DeferredReadManager::kickReads(int const) STUB
 
 #include "comm.h"
 bool comm_iocallbackpending(void) STUB_RETVAL(false)
-int commSetNonBlocking(int fd) STUB_RETVAL(Comm::COMM_ERROR)
-int commUnsetNonBlocking(int fd) STUB_RETVAL(-1)
-void commSetCloseOnExec(int fd) STUB_NOP
-void commSetTcpKeepalive(int fd, int idle, int interval, int timeout) STUB
-void _comm_close(int fd, char const *file, int line) STUB
-void old_comm_reset_close(int fd) STUB
-void comm_reset_close(const Comm::ConnectionPointer &conn) STUB
-int comm_connect_addr(int sock, const Ip::Address &addr) STUB_RETVAL(-1)
+int commSetNonBlocking(int) STUB_RETVAL(Comm::COMM_ERROR)
+int commUnsetNonBlocking(int) STUB_RETVAL(-1)
+void commSetCloseOnExec(int) STUB_NOP
+void commSetTcpKeepalive(int, int, int, int) STUB
+void _comm_close(int, char const *, int) STUB
+void old_comm_reset_close(int) STUB
+void comm_reset_close(const Comm::ConnectionPointer &) STUB
+#if LINGERING_CLOSE
```

Comment:
Support for `LINGERING_CLOSE` was recently removed.

Reply:
removed